_rules/css-orientation-lock-media-queries-b33eff.md

```diff
@@ -0,0 +1,199 @@
+---
+id: b33eff
+name: Page has no orientation lock specified using CSS media queries
+rule_type: atomic
+description: |
+  This rule checks that page content is not locked to any single display orientation using CSS media queries.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any element that is [visible](#visible) and has a CSS `transform` property with a [rotate](https://drafts.csswg.org/css-transforms/#funcdef-transform-rotate) transform-function, set from a CSS media query with an [orientation](https://drafts.csswg.org/mediaqueries-3/#orientation) media feature (`landscape` or `portrait`).
+
+## Expectation 1
+
+Each target element is not restricted to a single display orientation using CSS rotate transform-function, whose value is a multiple of 90 degrees but not a multiple of 180 degrees.
+
+## Assumptions
+
+- The rule does not consider elements for which a particular display orientation is [essential](https://www.w3.org/TR/WCAG21/#dfn-essential).
+- The rule does not consider the existence of any control on the page that can change the orientation on demand.
+
+## Accessibility Support
+
+*There are no major accessibility support issues known for this rule.*
+
+## Background
+
+- [Understanding Success Criterion 1.3.4: Orientation](https://www.w3.org/WAI/WCAG21/Understanding/orientation.html)
+- [Managing screen orientation](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model/Managing_screen_orientation)
+- [MDN Orientation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/orientation)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+A page, where CSS media query styles does not lock orientation. The CSS rotate transform-function value is a multiple of 180 degrees.
+
+```html
+<html lang="en">
+  <head>
+    <style>
+      @media (orientation: portrait) {
+        body {
+          transform: rotate(-360deg);
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <main>
+      Page Content
+    </main>
+  </body>
+</html>
+```
+
+#### Passed Example 2
```

Comment:
I've tried this on different browsers on iOS and I wasn't able to get the content right side up on portrait orientation. It was always flipped upside down. It's true that the content is not orientation locked (I can see it fine in landscape), but this is not a nice, accessible example.